dts/bindings/i2c/atmel,sam-i2c-twim.yaml

```diff
@@ -0,0 +1,183 @@
+# Copyright (c) 2020 Gerson Fernando Budke <nandojve@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: Atmel SAM Family I2C (TWIM) node
+
+compatible: "atmel,sam-i2c-twim"
+
+include: i2c-controller.yaml
+
+properties:
+    reg:
+      required: true
+
+    interrupts:
+      required: true
+
+    peripheral-id:
+      type: int
+      required: true
+      description: peripheral ID
+
+    std-clk-slew-lim:
+      type: int
+      required: true
+      description: |
+        Slew limit of the TWCK output buffer.  This should be adjusted with
+        std-clk-strength-low to fine tune the TWCK slope.
```

Comment:
There are a lot of parameters here; it'd be nice to provide a clue to people configuring a new board.  Understanding that they're inter-related, is it reasonable to expect the power-up configuration to be basically functional?  If so, documenting what that would be would help people get started without having to pull out a scope.

Reply:
Added description to address basic functionality.